Mathlib/Analysis/SpecialFunctions/ExpDeriv.lean

```diff
@@ -43,13 +43,20 @@ theorem analyticOnNhd_cexp : AnalyticOnNhd ℂ exp univ := by
 theorem analyticOn_cexp : AnalyticOn ℂ exp univ := analyticOnNhd_cexp.analyticOn
 
 /-- `exp` is analytic at any point -/
+@[fun_prop]
 theorem analyticAt_cexp : AnalyticAt ℂ exp z :=
   analyticOnNhd_cexp z (mem_univ _)
 
 /-- `exp ∘ f` is analytic -/
+@[fun_prop]
 theorem AnalyticAt.cexp (fa : AnalyticAt ℂ f x) : AnalyticAt ℂ (fun z ↦ exp (f z)) x :=
   analyticAt_cexp.comp fa
 
+/-- `exp ∘ f` is analytic -/
+@[fun_prop]
+theorem AnalyticAt.cexp' (fa : AnalyticAt ℂ f x) : AnalyticAt ℂ (exp ∘ f) x :=
```

Comment:
I'd exchange the primed and non-primed version, for coherence with the other formulations in the PR.

Reply:
Args. Fixed according to your suggestion.